src/OrbitQt/MoveFilesDialog.cpp

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "OrbitQt/MoveFilesDialog.h"
+
+#include <QLabel>
+#include <QPlainTextEdit>
+#include <QScrollBar>
```

Comment:
Is QLabel and QScrollbar used in the code below? I can't spot them. But QString should probably be here.